iterators.tex

```diff
@@ -6669,6 +6669,22 @@
 @\newtxt{\}}@
 \end{codeblock}
 
+{\color{newclr}
+\rSec3[ranges.output]{Output ranges}
+
+\pnum
+The \tcode{OutputRange} concept describes requirements of an \tcode{Range} type
+for which \tcode{begin} returns a type that satisfies
+\tcode{OutputIterator}~(\ref{output.iterators}).
+
+\begin{codeblock}
+template <class T>
+concept bool OutputRange() {
+  return Range<T>() && OutputIterator<IteratorType<T>>();
```

Comment:
Yes, of course - as should be obvious to anyone who is paying attention. Fixed and rebased onto master.